entrypoint.sh

```diff
@@ -103,7 +103,11 @@ if [[ -n ${AIRFLOW__CELERY__BROKER_URL} ]] && \
 fi
 
 if [[ ${AIRFLOW_COMMAND} == "bash" ]]; then
-   exec "/bin/bash"
+   shift
+   exec "/bin/bash" "${@}"
+elif [[ ${AIRFLOW_COMMAND} == "python" ]]; then
+   shift
+   exec "python" "${@}"
```

Comment:
```suggestion
elif [[ ${AIRFLOW_COMMAND} == "python"* ]]; then
   shift
   exec "${AIRFLOW_COMMAND}'" "${@}"
```

That way it would work with `python`, `python3` etc. (Haven't checked my syntax)

Reply:
I don't think there is a need. The image to choose from 'python2.7, 3.5, 3.6.... " determines what's the python version used. I think we should always use "python" and we get the right version automagically.